app/views/scaladoc.scala.html

```diff
@@ -23,7 +23,7 @@
 <h1>ScalaTest's Scaladoc documentation</h1>
 
 <ul>
-<li>Scaladoc for 3.0.1 - 3.0.4 is coming soon. Please use the Scaladoc for 3.0.0 for these versions in the meantime.</li>
+<li>Scaladoc for 3.0.1 - 3.0.5 is coming soon. Please use the Scaladoc for 3.0.0 for these versions in the meantime.</li>
```

Comment:
The Scaladoc for 3.0.1 is published. Perhaps this line should be updated to reflect this (and the following line uncommented).